frontends/bmd/src/app_mark_voter_card_invalid.test.tsx

```diff
@@ -179,48 +171,4 @@ describe('Mark Card Void when voter is idle too long', () => {
     await advanceTimersAndPromises();
     screen.getByText('Insert voter card to load ballot.');
   });
-
-  test('Reset ballot when card write does not match card read.', async () => {
```

Comment:
We no longer read the card after writing to check that it worked. I didn't think it was necessary.

Reply:
Insurance against a broken card actually. This does happen from time to time. Less likely to be a problem when we don't have voter cards, but a bit of a risk when we have them.

Reply:
I think it would be wise to keep it for now yes.

Reply:
Ok I added the test back in and it actually still passes, since there is a check to ensure that the short value on the card is well-formed user data. Do you think that's enough protection against broken cards?